pkg/api/validation/validation.go

```diff
@@ -762,6 +762,9 @@ func validateHTTPGetAction(http *api.HTTPGetAction) errs.ValidationErrorList {
 	} else if http.Port.Kind == util.IntstrString && len(http.Port.StrVal) == 0 {
 		allErrors = append(allErrors, errs.NewFieldRequired("port"))
 	}
+	if http.Scheme != api.URISchemeHTTP || http.Scheme != api.URISchemeHTTPS {
+		allErrors = append(allErrors, errs.NewFieldInvalid("scheme", http.Scheme, "must be a supported scheme for probing"))
```

Comment:
just have a `supportedGetActionSchemes` list you do a `.Has()` on for validating and a `strings.Join(....List(), ", "))` on for the message